### ISSUE TITLE
I'm feeling lucky on the page

### ISSUE THREAD

**Issue by titipata** [titipata/awesome-thai-girls#4]
@bluenex Can you add `I'm feeling lucky` link to the page? Here is Javascript to randomly pick one Instagram from the page.

```
var links = [], l = document.links;
for(var i=0; i<l.length; i++) {
  if (l[i].href.includes("instagram") == true) {
  links.push(l[i].href);
  }
}

var feeling_lucky = links[Math.floor(Math.random() * links.length)];
```

**Comment by titipata** [comment]
- getting all the href elements: http://stackoverflow.com/questions/3871358/get-all-the-href-attributes-of-a-web-site
- getting random choice from array: http://stackoverflow.com/questions/4550505/getting-random-value-from-an-array

**Comment by titipata** [comment]
closed by 887704a59451fc069a5e13c50767573477057ec9
